libr/arch/p/pyc/plugin.c

```diff
@@ -171,36 +155,35 @@ static int pyc_op(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *data, int len, RA
 	anal_pyc_op (op, op_obj, oparg);
 
 anal_end:
-	//free_opcode (ops);
+	// free_opcode (ops);
```

Comment:
Can be removed because of finish callback?
```suggestion
```

Reply:
thanks